### ISSUE TITLE
[Feature request] Per-game plugin activation (Adrenaline)

### ISSUE THREAD

**Issue by rafaelferreira01** [cy33hc/vita-launcher#36]
On a real PSP (or Adrenaline) we can use a plugin called [Pergame](https://www.gamebrew.org/wiki/Pergame_Custom_PSP) to activate one or more plugins, depending on game.
With this plugin i can, for instance, activate the CWCheat plugin only while playing GTAVCS, and while playing other games the plugin will remain disabled.

I use Pergame with Adrenaline to launch PSP games with the working GePatch version and some dual analog plugins, but if i launch the game though vita-launcher it doesn't work (i think the plugin takes action while booting the game). 

Is there a way to implement something like this?

**Comment by rafaelferreira01** [comment]
Idea:
Allow us to create a "game.txt" in the game folder. This file must contain only the specific plugins activated.

Like this:
`ms0:/seplugins/remasteredcontrols/tonyhawk_remastered.prx.prx 1`

While my the default game.txt  (the one in pspemu\seplugins\) look like this:
```
ms0:/seplugins/cwcheat/cwcheat.prx 1
ms0:/seplugins/tempar/tempar.prx 0
ms0:/seplugins/atpro.prx 1
```
So, when launching a PSP game vita-launcher need to search for a "game.txt" in the game folder, if this file doesn't exists the game is launched with the default game.txt but if it exists then it's merged with the default game.txt (in some temp folder), ending like this:

```
ms0:/seplugins/remasteredcontrols/tonyhawk_remastered.prx.prx 1
ms0:/seplugins/cwcheat/cwcheat.prx 1
ms0:/seplugins/tempar/tempar.prx 0
ms0:/seplugins/atpro.prx 1
```

Then the game is launched using this file instead of the default one.

**Comment by cy33hc** [comment]
@rafaelferreira01 When you boot a PSP game, there is a Boot settings screen when you can enable "Plugins". Have you try enabling it?

**Comment by rafaelferreira01** [comment]
@cy33hc Just tested with a couple of plugins. Pergame is the only plugin that won't work when launching a game from vita-launch.

**Comment by cy33hc** [comment]
@rafaelferreira01 Can you try it with a different driver like NP9660 or March33

**Comment by rafaelferreira01** [comment]
@cy33hc None of them worked.

**Comment by cy33hc** [comment]
@rafaelferreira01 I'm starting to work on this feature, but I don't use plugins at all. I know in the PSP, there is a game.txt for PSP ISO games, also there is a pops.txt for PS1 (EBOOT) games. Does pops.txt exists in adrenaline? Also does the game.txt plugins apply to PSP Minis too?

**Comment by rafaelferreira01** [comment]
@cy33hc Nice! Yes, pops.txt exists and game.txt affects PSP Minis too.

**Comment by cy33hc** [comment]
@rafaelferreira01 This is implemented in the latest release.

**Comment by rafaelferreira01** [comment]
You're the man, it's working like a charm! There's a couple of things I noticed though:
1. When we made some change on the pergame plugins (enabling or disabling) and press Ok to launch the game, AdrenalineLauncher tries to launch the game without success.
2. About the plugin list (global or pergame). When we press down the cursor always go to below delete button and when we press up it goes to the above checkbox.
